benchmark/single-source/StringTests.swift

```diff
@@ -19,6 +19,7 @@ public func run_StringWithCString(_ N: Int) {
 }
 
 public func run_StringHasPrefix(_ N: Int) {
```

Comment:
Can you add a `// FIXME(string)` for these ones? This particular function ought to get added for Linux as part of the String rework, so hopefully we can restore this test on Linux then (unlike some of the objc runtime tests which may only ever run on Darwin).